pandas/tests/indexes/test_multi.py

```diff
@@ -2535,3 +2536,19 @@ def test_dropna(self):
         msg = "invalid how option: xxx"
         with tm.assertRaisesRegexp(ValueError, msg):
             idx.dropna(how='xxx')
+
+    def test_unsortedindex(self):
+        # GH 11897
+        mi = pd.MultiIndex.from_tuples([('z', 'a'), ('x', 'a'), ('y', 'b'),
```

Comment:
we want to find places where currently a KeyError is raised and change them in the tests.